src/components/CardSelector/CardSelector.tsx

```diff
@@ -114,16 +116,15 @@ export const CardSelector = React.forwardRef<HTMLDivElement, CardSelectorProps>(
                 </div>
                 {(chips || information) && (
                     <div className={clsx('flexRow_nowrap')}>
-                        {chips && (
-                            chips.map(chip => (chip))
-                        )}
+                        {chips?.map(chip => (chip))}
```

Comment:
I believe we can  just render this directly if it's already a list of react components (assuming there's already a `key` param added to the components), no need for the map:

```suggestion
                        {chips}
```

Reply:
Right, simpler is better !